py/torch_tensorrt/dynamo/conversion/impl/condition/ops.py

```diff
@@ -10,8 +9,7 @@
     broadcastable,
     get_trt_tensor,
 )
-from torch_tensorrt.dynamo.conversion.impl.slice import expand
-from torch_tensorrt.fx.converters.converter_utils import set_layer_name
+from torch_tensorrt.fx.converters.converter_utils import prepend_ones, set_layer_name
```

Comment:
can we add prepend_ones and the set_layer_name functions in our dynamo codebase instead of using from fx ?

Reply:
I moved it to py/torch_tensorrt/dynamo/conversion/converter_utils.py